backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/database/Schema.kt

```diff
@@ -14,13 +14,15 @@ import org.jetbrains.exposed.sql.or
 
 const val CARD_INFO_HASH_LENGTH = 32 // Using SHA256-HMAC
 const val TOTP_SECRET_LENGTH = 20
+const val ACTIVATION_SECRET_LENGTH = 20
```

Comment:
@maxammann Please suggest some length :P

Reply:
Definitely reasonable 👌🏼 

It's like a password with 160bit entropy, which is comparable to a 25 char PW.